<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDomeInfo</class>
 <widget class="QGroupBox" name="QDomeInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>1041</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dome</string>
  </property>
  <property name="title">
   <string>Dome</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="DomeWidget" name="picture" native="true"/>
   </item>
   <item>
    <widget class="QFloatLine" name="dl_time_alive" native="true"/>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_cover">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Cover</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="lb_cover_state">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>undefined</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_cover_comment">
        <property name="text">
         <string>undefined</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="pb_cover">
        <property name="maximum">
         <number>401</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="format">
         <string>%v/%m</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QBooleanLine" name="bl_servo_moving" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QBooleanLine" name="bl_servo_direction" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QBooleanLine" name="bl_open_dome_sensor" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QBooleanLine" name="bl_closed_dome_sensor" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QBooleanLine" name="bl_safety" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QBooleanLine" name="bl_servo_blocked" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hl_openclose">
        <item>
         <widget class="QPushButton" name="bt_cover_open">
          <property name="text">
           <string>open</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bt_cover_close">
          <property name="text">
           <string>close</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_sensors">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Sensors</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QFloatLine" name="bl_t_lens" native="true"/>
      </item>
      <item>
       <widget class="QFloatLine" name="bl_t_CPU" native="true"/>
      </item>
      <item>
       <widget class="QFloatLine" name="bl_t_SHT31" native="true"/>
      </item>
      <item>
       <widget class="QFloatLine" name="bl_h_SHT31" native="true"/>
      </item>
      <item>
       <widget class="QBooleanLine" name="bl_rain_sensor" native="true"/>
      </item>
      <item>
       <widget class="QBooleanLine" name="bl_light_sensor" native="true"/>
      </item>
      <item>
       <widget class="QBooleanLine" name="bl_master_power" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_devices">
     <property name="title">
      <string>Devices</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QControlLine" name="cl_lens_heating" native="true"/>
      </item>
      <item>
       <widget class="QControlLine" name="cl_camera_heating" native="true"/>
      </item>
      <item>
       <widget class="QControlLine" name="cl_fan" native="true"/>
      </item>
      <item>
       <widget class="QControlLine" name="cl_ii" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_errors">
     <property name="title">
      <string>Error bits</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QBooleanLine" name="bl_error_t_lens" native="true"/>
      </item>
      <item>
       <widget class="QBooleanLine" name="bl_error_t_SHT31" native="true"/>
      </item>
      <item>
       <widget class="QBooleanLine" name="bl_error_light" native="true"/>
      </item>
      <item>
       <widget class="QBooleanLine" name="bl_error_watchdog" native="true"/>
      </item>
      <item>
       <widget class="QBooleanLine" name="bl_error_brownout" native="true"/>
      </item>
      <item>
       <widget class="QBooleanLine" name="bl_error_master" native="true"/>
      </item>
      <item>
       <widget class="QBooleanLine" name="bl_error_t_CPU" native="true"/>
      </item>
      <item>
       <widget class="QBooleanLine" name="bl_error_rain" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFloatLine</class>
   <extends>QWidget</extends>
   <header>widgets/lines/qfloatline.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DomeWidget</class>
   <extends>QWidget</extends>
   <header>widgets/domewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QBooleanLine</class>
   <extends>QWidget</extends>
   <header>widgets/lines/qbooleanline.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QControlLine</class>
   <extends>QWidget</extends>
   <header>widgets/lines/qcontrolline.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
